From d158e533f9e735ef268d69c3fb4e03e233d69bf9 Mon Sep 17 00:00:00 2001
From: Vitaly Lipatov <lav@etersoft.ru>
Date: Thu, 3 Oct 2013 14:41:16 +0400
Subject: [PATCH 24/24] 0.16-eter25.svn724
To: wine-patches <wine-patches@winehq.org>
Reply-To: wine-devel <wine-devel@winehq.org>

- remove PidFile option from cups config (eterbug #9490)
---
 .gear/opennx.spec | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/.gear/opennx.spec b/.gear/opennx.spec
index d935c95..3d71a97 100644
--- a/.gear/opennx.spec
+++ b/.gear/opennx.spec
@@ -3,7 +3,7 @@
 
 Name: opennx
 Version: 0.16
-Release: eter24.svn724
+Release: eter25.svn724
 
 Summary: An OpenSource NX client
 
@@ -106,6 +106,9 @@ install -m 644 etc/*.rules %buildroot%_sysconfdir/udev/rules.d
 %endif
 
 %changelog
+* Thu Oct 03 2013 Vitaly Lipatov <lav@altlinux.ru> 0.16-eter25.svn724
+- remove PidFile option from cups config (eterbug #9490)
+
 * Mon Aug 12 2013 Vitaly Lipatov <lav@altlinux.ru> 0.16-eter24.svn724
 - use absolute path for links, fix requires
 
-- 
1.8.3.4

